pkg/apis/kops/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -128,6 +128,15 @@ func (in *Assets) DeepCopyInto(out *Assets) {
 			**out = **in
 		}
 	}
+	if in.ContainerProxy != nil {
```

Comment:
Why is this included here? Doesn't seem related to failing the test

Reply:
It’s the fix to have tests not fail anymore. But could be split out to another pull request as well.

Reply:
Discussed in chat. No worries!